devenv/sync.py

```diff
@@ -79,12 +80,20 @@ def run_procs(
 
 def main(context: Dict[str, str], argv: Sequence[str] | None = None) -> int:
     repo = context["repo"]
-    if repo not in {"sentry", "getsentry"}:
-        print(f"repo {repo} not supported yet!")
-        return 1
-
     reporoot = context["reporoot"]
 
+    if os.path.exists(f"{reporoot}/devenv/sync.py"):
+        spec = importlib.util.spec_from_file_location(
+            "sync", f"{reporoot}/devenv/sync.py"
+        )
+        module = importlib.util.module_from_spec(spec)  # type: ignore
+        spec.loader.exec_module(module)  # type: ignore
+        return module.main(context)  # type: ignore
+
+    # past this point is legacy code that should only be run for sentry/getsentry
+    # eventually to be moved to sentry/devenv/sync.py
+    assert repo in {"sentry", "getsentry"}
+
```

Comment:
Add a TODO or FIXME so we can more easily remember to come back to this?

Reply:
nah, i plan on doing this once it's shipped